dashboard/test/controllers/projects_controller_test.rb

```diff
@@ -248,4 +248,17 @@ def sign_in_with_request(user)
     get :load, params: {key: 'gamelab'}
     assert_redirected_to_sign_in
   end
+
+  test 'project validators can go to /featured' do
+    @project_validator = create :teacher
```

Comment:
does a project validator need to be a teacher?

Reply:
I think permissions are only for teachers.  Do we ever give permissions to students? Specifically I used teacher in this case because https://github.com/code-dot-org/code-dot-org/blob/26743f115045dfb424eca34a3356fc4bd9f04a2d/dashboard/app/models/user.rb#L296; unless I use teacher my check for project_validator permission in `user.rb` will return false.

Reply:
interesting, today I learned.